### ISSUE TITLE
Is there any plan to update this project to work with the newest version of kafka-python (1.3.0) library? 

### ISSUE THREAD

**Issue by damienkilgannon** [taykey/python-kafka-logging#12]

**Comment by petermoore14** [comment]
+1; causing me serious grief atm

**Comment by damienkilgannon** [comment]
yes, would be great to get pull request #13 merged

**Comment by avihoo** [comment]
You can check my fork of the library, as I originally wrote this library but no longer works at Taykey. I forked the library and updated the client to work with the new version.

You can check this out [here](https://github.com/avihoo/python-kafka-logging/)

**Comment by petermoore14** [comment]
Excellent; thanks!

**Comment by petermoore14** [comment]
Seems to be working very well; are there any plans to register your fork on PyPi?

**Comment by avihoo** [comment]
Yes. Just did that. You can now install it using `pip` by typing:
`pip install python-kafka-logger`

Version `0.2` is the most recent one.

**Comment by rafmim** [comment]
Avihoo - u rock!
